### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by Lukkee-1** [Mqlvin/CoralClient#1]
Can you make a tutorial / more dertailed readme on how to use the plugin / installing? Please and thank you.

**Comment by Mqlvin** [comment]
Hello @Lukkee-1

Unfortunately I never got around to finishing this project, and the client literally only has two hack modules right now. It is in a test/buggy state however if you still would like to test this client, I can compile it for you and explain how to install that.

Thanks for taking interest in this, and sorry it wasn't complete,
Mqlvin

**Comment by Lukkee-1** [comment]
> Hello @Lukkee-1
> 
> Unfortunately I never got around to finishing this project, and the client literally only has two hack modules right now. It is in a test/buggy state however if you still would like to test this client, I can compile it for you and explain how to install that.
> 
> Thanks for taking interest in this, and sorry it wasn't complete, Mqlvin

yeah I can definitely test it for you . DM me on discord with info it will be easier
My name is Luke.#6942

**Comment by Mqlvin** [comment]
I mean it's a really incomplete client and I don't plan to work on it to be honest. I have still sent a friend request on Discord and I can help you set it up.

Not sure if I cqn help right now though, as I don't have access to my PC, so I may not be able to compile it for a few days.
